plugins/eddn.py

```diff
@@ -635,6 +638,9 @@ def export_commodities(self, data: CAPIData, is_beta: bool) -> None:  # noqa: CC
         :param data: a dict containing the starport data
         :param is_beta: whether or not we're currently in beta mode
         """
+        should_return: bool
+        new_data: Dict[str, Any] = {}
```

Comment:
```suggestion
        new_data: dict[str, Any]
```

There are more of these, but Im not going to cover each one :sweat_smile:

Reply:
Ok, I've changed all that I originally added in this PR, but there are many more across the codebase and I don't want to expand the footprint of my PR any further - these should be fixed in a separate PR IMHO.